app/controllers/filters_controller.rb

```diff
@@ -66,7 +66,7 @@ def role_id
   def setup_search_options
     @original_search_parameter = params[:search]
     params[:search] ||= ""
-    params.each do |param, value|
+    params.each_key do |param|
```

Comment:
`value` is used on lines 71 & 76 so this is a false positive?

Reply:
I guess it would be better to revert to `params.each` to handle both `key` and `value` in this case.